### ISSUE TITLE
Test for defaults failing

### ISSUE THREAD

**Issue by graysonhicks** [graysonhicks/gatsby-plugin-remote-images#19]
@smurrayatwork Ran the test suite and getting this failure.  For some reason `fileNode.url` is returning `undefined` in the default test.

``` FAIL  src/__tests__/gatsby-node.js
  gatsby-plugin-remote-images
    ✕ creates remote file node with defaults (257ms)
    ✓ can use the name option (1ms)
    ✓ can use the ext option (169ms)

  ● gatsby-plugin-remote-images › creates remote file node with defaults

    expect(received).toBe(expected) // Object.is equality

    Expected: "https://dummyimage.com/600x400/000/fff.png"
    Received: undefined

      42 |       expect(createNodeId).toHaveBeenCalledTimes(1);
      43 |       expect(fileNode.ext).toBe('.png');
    > 44 |       expect(fileNode.url).toBe(node.imageUrl);
         |                            ^
      45 |       expect(fileNode.internal.type).toBe('File');
      46 |       expect(fileNode.parent).toBe('testing');
      47 |       expect(node['localImage___NODE']).toBe('remoteFileIdHere');

      at toBe (src/__tests__/gatsby-node.js:44:28)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 2 passed, 3 total
Snapshots:   0 total
Time:        1.782s, estimated 2s
Ran all test suites.```

**Comment by graysonhicks** [comment]
Should be solved by #21

**Comment by graysonhicks** [comment]
Fixed by #21
